code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -14,6 +14,7 @@
 	var/buildstacktype = /obj/item/stack/sheet/iron
 	var/buildstackamount = 1
 	var/item_chair = /obj/item/chair // if null it can't be picked up
+	var/obj/item/assembly/shock_kit/stored_kit
```

Comment:
Document, but why is this on chair anyway if it's supposed to be a component?

Reply:
no longer exists, the item is now put into parent by the component